### PR TITLE
Page de confirmation d'inscription à la newsletter

### PR DESCRIPTION
**Pull request by AntoineAresu**
[Ticket](https://trello.com/c/goFT7WH4/3641-3-page-de-confirmation-dinscription-%C3%A0-la-newsletter)

### CODE REVIEW

File: templates/homeV2/sections/_newsletter_confirmation_section.html.twig [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    },
+    {
+        url: 'https://medium.com/reconnect',
+        icon: 'fa-medium-m',
+    },
+] %}
+
+<section class="mt-3 py-3 pt-md-5 bg-secondary">
+    <div class="container text-center text-white mt-5">
+        <h2 class="h2 pt-5 pb-5 text-uppercase"><b>{{ 'thank_you_for_subscription'|trans }}</b></h2>
```
Comment (tcheymol) [inline templates/homeV2/sections/_newsletter_confirmation_section.html.twig:26]: do we really need to add the class h2 on a h2  ?

File: templates/homeV2/pages/components/_reconnect_information.html.twig [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="my-3 my-md-0">
```
[the remark this answers is not in the document]
Comment (AntoineAresu) [reply templates/homeV2/pages/components/_reconnect_information.html.twig:1]: it doesn't work this way

File: translations/messages.en.yml [hunk 3]
```diff
@@ -895,6 +899,7 @@ switchUser.connecte: 'You are connected as <span class="italic">%username%'
 username: Username
 tax: Tax
 terms_of_use: Terms of use
+thank_you_for_subscription: Thank you for your subscription
```
Comment (tcheymol) [inline translations/messages.en.yml:902]: Thanks for subscribing

File: templates/homeV2/pages/newsletter_confirmation.html.twig [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{% extends 'homeV2/base.html.twig' %}
+
+{% block title %}{{ 'subscription_confirmation'|trans }}{% endblock %}
+{% block meta_title %}{{ 'subscription_confirmation'|trans }}{% endblock %}
+{% block metas %}<meta name="robots" content="noindex">{% endblock %}
+
+{% block body %}
+    {{ include('homeV2/sections/_newsletter_confirmation_section.html.twig') }}
+    {{ include('homeV2/sections/our-solutions-section.html.twig', {
```
[the remark this answers is not in the document]
Comment (AntoineAresu) [reply templates/homeV2/pages/newsletter_confirmation.html.twig:9]: yes

File: templates/homeV2/sections/_newsletter_contact_us_section.html.twig [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<section class="mt-3 py-3 pt-md-5 bg-primary">
+    <div class="row text-center text-white">
```
[the remark this answers is not in the document]
Comment (AntoineAresu) [reply templates/homeV2/sections/_newsletter_contact_us_section.html.twig]: we need to override text color on link

File: templates/homeV2/pages/components/_reconnect_information.html.twig [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+    <p class="font-size-1-3 my-0">Reconnect</p>
```
[the remark this answers is not in the document]
Comment (AntoineAresu) [reply templates/homeV2/pages/components/_reconnect_information.html.twig:2]: we need it

File: translations/messages.fr.yml [hunk 7]
```diff
@@ -967,6 +972,10 @@ quitterCentre.titre: 'Quitter le centre %s'
 recommonbundle.uploadPhoto: 'Cliquez ici pour choisir une image'
 reconnect_center: 'Relais Reconnect'
 reconnect_center_span: 'Relais <span class="rp-blue">Reconnect</span>'
+reconnect_contact_mail: contact@reconnect.fr
+reconnect_phone_number: 01 55 28 95 33
+reconnect_rgpd_mail: rgpd@reconnect.fr
+rediscover_our_solutions: (re)Découvrez nos solutions
 regions: Régions
 registerForm.adresse: 'Votre adresse personnelle (facultatif)'
```
[the remark this answers is not in the document]
Comment (AntoineAresu) [reply translations/messages.fr.yml:980]: The text comes from the model that Roxane gave me

File: translations/messages.fr.yml [hunk 8]
```diff
@@ -1138,23 +1152,16 @@ website_CGS: CGS du site
 website_confidentiality: Politique de confidentialité du site
 website_legal: Mentions légales
 website_title: 'Reconnect'
+while_waiting_next_newsletter_1: En attendant la prochaine, vous pouvez
+while_waiting_next_newsletter_2: retrouver nos actualités sur nos réseaux sociaux et notre blog !
 work: Emploi
 wrong_current_password: 'Votre mot de passe actuel est faux'
 wrong_secret_answer: La réponse à la question secrète est incorrecte
 you_must_provide_phone_number: Vous devez renseigner le numéro de téléphone
+you_are_now_registered_for_newsletter: Vous êtes maintenant inscrit.e sur notre liste d'envoi pour recevoir nos newsletter
```
[the remark this answers is not in the document]
Comment (AntoineAresu) [reply translations/messages.fr.yml:1161]: same, the text comes from the model

**Review by tcheymol — APPROVED** [review]

**Pull request merged**